backend/app/routes/Q&A/question/upvoteQuestion.js

```diff
@@ -12,10 +13,11 @@ module.exports = async (req, res, next) => {
       message: 'Database Error',
       errStack: err,
     });
-
     return next(error);
   }
 
+  res.cookie(getVoteCookieName('question', questionId), true, { maxAge: 20 * 365 * 24 * 60 * 60 * 1000 });
```

Comment:
what is the unit of this?

Reply:
I kept it to 20 years. why because no one can use a mobile for 20 years.